BaseTools/Source/C/Makefiles/ms.common

```diff
@@ -44,13 +44,23 @@ LIB_PATH     = $(BASE_TOOLS_PATH)\Lib\Win64
 SYS_BIN_PATH = $(EDK_TOOLS_PATH)\Bin\Win64
 SYS_LIB_PATH = $(EDK_TOOLS_PATH)\Lib\Win64
 
-# MU_CHANGE Starts: Adding support to build base tools for ARM
+# MU_CHANGE Starts: Adding support to build base tools for ARM and AARCH64
 !ELSEIF "$(HOST_ARCH)"=="ARM"
 ARCH_INCLUDE = $(SOURCE_PATH)\Include\Arm
 BIN_PATH     = $(BASE_TOOLS_PATH)\Bin\Win32
 LIB_PATH     = $(BASE_TOOLS_PATH)\Lib\Win32
 SYS_BIN_PATH = $(EDK_TOOLS_PATH)\Bin\Win32
 SYS_LIB_PATH = $(EDK_TOOLS_PATH)\Lib\Win32
+!ELSEIF "$(HOST_ARCH)"=="AARCH64"
+ARCH_INCLUDE = $(SOURCE_PATH)\Include\AArch64
+BIN_PATH     = $(BASE_TOOLS_PATH)\Bin\Win64
+LIB_PATH     = $(BASE_TOOLS_PATH)\Lib\Win64
+SYS_BIN_PATH = $(EDK_TOOLS_PATH)\Bin\Win64
+SYS_LIB_PATH = $(EDK_TOOLS_PATH)\Lib\Win64
+# Note: These are some temporary warning suppression we carry to avoid much more overrides
+#       The expected outcome is that all of these C file based tool will be replaced by py
+#       scripts, when it happens..
+CFLAGS = $(CFLAGS) /wd4267 /wd4244 /wd4334

```

Comment:
Nope, these are the warning encountered when building 64-bit windows applications. All 3 warnings suppressed are conversion related:
https://learn.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-2-c4244?view=msvc-170
https://learn.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-3-c4267?view=msvc-170
https://learn.microsoft.com/en-us/cpp/error-messages/compiler-warnings/compiler-warning-level-3-c4334?view=msvc-170

The offline discussion agreed on moving to python scripts when EDK2 has them, which is why I put temporary here :) It might be far out though. I can change the wording to clarify that.

Reply:
I think a comment might be helpful when someone else encounters the warnings as well, but not strictly necessary since there is a future path.

Reply:
Comment updated.

Reply:
Comment updated.